src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.util.Set;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String TITLES = "fruit,quantity";
+    private static final String COMMA_SEPARATOR = ",";
+
+    @Override
+    public String createReport() {
+        Set<String> fruits = Storage.getStorage().keySet();
+        StringBuilder builder = new StringBuilder(TITLES);
```

Comment:
do not use such naming, how to guess what it stores?
```suggestion
        StringBuilder report = new StringBuilder(TITLES);
```